jfrog_platform_cloud/manifest.json

```
@@ -0,0 +1,52 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "798102cb-6c52-4a16-bc1b-48c2e6b54e71",
+  "app_id": "jfrog-platform-cloud",
+  "display_on_public_website": false,
```

Will want this to be `true` to publish this as a docs page.

```suggestion
  "display_on_public_website": true,
```